envs.json

```diff
@@ -142,7 +142,7 @@
     },
     "solana_url": "http://127.0.0.1:8899/",
     "faucet_url": "http://127.0.0.1:3333/",
-    "tracer_url": "http://neon-rpc:9090",
+    "tracer_url": "http://127.0.0.1:9091",
```

Comment:
Did you ask @romanova-natasha about this?

Reply:
Yes!